### DISCUSSION TITLE
Invalid sparse file format at header magic

### DISCUSSION THREAD

**Discussion by galosre** [tesla-android/issue-tracker#69] [Q&A]
Got this issue making the SD card multiplace got "Invalid sparse file format at header magic" from a window or a mac I just tried again today no matter what it can write the SD but  on the screen once Android boot
always get There is an internal problem in your device. Please contact your manufacturer.
 And I have to press OK for the warning to go away
[Invalid sparse file format at header magic.txt](https://github.com/tesla-android/issue-tracker/files/9080404/Invalid.sparse.file.format.at.header.magic.txt)
?

**Comment by bstmedia** [comment]
@galosre  I see the same but this does not seems to impact the installation. My first attempt failed because of outdated "fastboot" version. I would suggest that you download the latest version and start all over. Make sure that all commands in step 8 are accepted and dont result in error.

**Comment by mikegapinski** [comment]
Invalid sparse file format at header magic - this message is not an error - you can ignore this

**Comment by galosre** [comment]
Glad to know but still getting internal problem I am sure something is not right there 
![internal_ problem](https://user-images.githubusercontent.com/22267106/178350397-06f74fea-2645-4035-8f07-4f919e200687.jpg)
?

**Comment by galosre** [comment]
Beside that now we are talking! I drove today to lax and basically everything was working very happy, not sure about the phone because my phone did not pair with the car so at home I had to reboot the car's computer. I will see tomorrow, but I am  sure it is ok. I also appreciate the fact we can now use some custom apps, and Also Now I can put my LTE sim card back inside my tablet, overall very good experience.

Anecdote 
Also, my PI3b c r@#$% on me, the power capacitor   became hot and  magic smoke was coming out!, so I had to use the one from my Stratux I do plan to use the Stratux  it for the next 2 months so maybe by then hopefully  android_Tesla may use only one PI4
